### DISCUSSION TITLE
Ajouter un bloc d'explication sur les frais annexe lors de l'embauche (dont la médecine du travail)

### DISCUSSION THREAD

**Discussion by mquandalle** [betagouv/mon-entreprise#2694] [Idées]
#1765 retire un modèle trop simplifié du prix de la médecine du travail.

Il faudrait ré-introduire cet élément en prenant mieux en compte les paramètres du calcul (nombre de salariés, périodicité des visites) et en ajoutant un bloc d'explication sur ce sujet.

Par ailleurs il faudrait élargir à tous les frais annexes à prendre en compte lors du recrutement d'un salarié (poste, matériel, local, uniforme, formation, ...)

**Comment by glopezrios** [comment]
Un usager regrette que le simulateur pour salarié ne prenne pas en compte le coût de la médecine du travail (même si c'est bien indiqué dans la page de documentation [du coût total employeur](https://mon-entreprise.urssaf.fr/documentation/salari%C3%A9/co%C3%BBt-total-employeur).

Je ne fais pas une issue spécifique car ce coût est calculé de manière très différente selon les entreprises. Une grande entreprise peut avoir un médecin salarié tandis que des petites entreprises peuvent cotiser à un organisme externe, un SPSTI.
